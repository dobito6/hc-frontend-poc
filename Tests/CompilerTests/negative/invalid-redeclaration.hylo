//! no-std

fun f(x: Void, y: Void, x: Void) {}

fun g(x: Void) {
  let x = ()
  let y = ()
  let y = ()
}
