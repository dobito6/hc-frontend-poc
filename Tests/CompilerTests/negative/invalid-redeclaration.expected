negative/invalid-redeclaration.hylo:3.25-26: error: invalid redeclaration of 'x'
fun f(x: Void, y: Void, x: Void) {}
                        ^
negative/invalid-redeclaration.hylo:3.7-8: note: previous declaration here
fun f(x: Void, y: Void, x: Void) {}
      ^
negative/invalid-redeclaration.hylo:6.7-8: error: invalid redeclaration of 'x'
  let x = ()
      ^
negative/invalid-redeclaration.hylo:5.7-8: note: previous declaration here
fun g(x: Void) {
      ^
negative/invalid-redeclaration.hylo:8.7-8: error: invalid redeclaration of 'y'
  let y = ()
      ^
negative/invalid-redeclaration.hylo:7.7-8: note: previous declaration here
  let y = ()
      ^
